ROADMAP.md

```diff
@@ -1,22 +1,46 @@
 # KF Serving Roadmap
 ## 2019 - 2020
+### v0.6 Inference Graph (ETA end of Q4)
+Objective: Enable model serving pipelines with flexible routing graph
+* Inference Router
+    * Model Experimentation.
+    * Ensembling.
+    * Multi Arm Bandit.
+    * Pipeline
+
+### v0.5 KFServing V1Beta1 API and V2 prediction API (ETA: end of Q3)
+Objective: Stable V1Beta1 InferenceService API!
+    * Promote v1alpha2 to v1beta1
+    * Conversion webhook
+
+Objective: Unify protocols across frameworks with [prediction V2 API](https://github.com/kubeflow/kfserving/tree/master/docs/predict-api/v2).
+    * V2 Python Server
+    * Triton inference server V2 prediction API
+    * TorchServe/KFServing integration
+
+Objective: "Reduce Total Cost of Ownership when deploying multiple underutilized models."
+* Container/GPU Sharing
+    * Reduce TCO by enabling models of the same framework and version to be co-hosted in a single model server.
 
-### v0.4 Performance (ETA: Jan 31, 2020)
+### v0.4 Performance (end of Q2)
```

Comment:
I'm not sure how much of this performance work was done. I know there was some ad hoc stuff. Should we punt on a Q2 release and just treat 0.4 as a kubeflow cut for kf 1.1?

Reply:
I have put up a PR for the detailed benchmarking https://github.com/kubeflow/kfserving/pull/894, we can call it a 0.4 release